pkg/db/db_client/db_client.go

```diff
@@ -30,6 +30,9 @@ type DbClient struct {
 	// connection used to run system/plumbing queries (connection state, server settings)
 	managementPool *pgxpool.Pool
 
+	// a connection hijacked from the pool before the pool is disabled
+	hijackedConnection *pgx.Conn
```

Comment:
rename var  and improve comment - why do we have a hijacked connection?
maybe call it `userConnection` ?